src/test/java/com/academia/andruhovich/library/util/JsonConvertHelper.java

```diff
@@ -0,0 +1,127 @@
+package com.academia.andruhovich.library.util;
+
+import com.academia.andruhovich.library.dto.AuthResponseDto;
+import com.academia.andruhovich.library.dto.AuthorDto;
+import com.academia.andruhovich.library.dto.BookDto;
+import com.academia.andruhovich.library.dto.OrderResponseDto;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import lombok.Data;
+
+import java.io.IOException;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.TimeZone;
+
+import static com.academia.andruhovich.library.exception.ErrorMessages.CANNOT_READ_JSON_VALUE;
+import static com.academia.andruhovich.library.util.DateHelper.CURRENT_LOCATION;
+
+@Data
+public class JsonConvertHelper {
+
+    private static final String CONTENT = "content";
+
+    private final static ObjectMapper objectMapper = new ObjectMapper()
```

Comment:
look same as  @Bean     public ObjectMapper ?   autoware

Reply:
Should i change this field on NOTstatic and all methods corse of @Autowired dont work with static?
![image](https://user-images.githubusercontent.com/53169096/147231058-a1063e3a-c660-458d-b67b-e7cbea9d7e62.png)

Reply:
Deleted JsonConvertHelper class